test/e2e/modal.test.js

```diff
@@ -2,7 +2,7 @@ const { assert, driver } = require('vl-ui-core').Test.Setup;
 const VlModalPage = require('./pages/vl-modal.page');
 const VlModalAutoOpenPage = require('./pages/vl-modal-auto-open.page');
 const { By } = require('selenium-webdriver');
-const { VlDatepicker } = require('vl-ui-datepicker').Test;
+// const { VlDatepicker } = require('vl-ui-datepicker').Test;
```

Comment:
Deze binnentrekken zorgde voor ophalen outdated dependency die nog niet automatisch de browsers afsloot.

Reply:
is er dan ergens geweten/genoteerd dat dit nog moet gebeuren bij de modal?

Reply:
Ticket staat nog op impeded hierdoor